### PR TITLE
add AMD 4004 series certification page

### PR DESCRIPTION
**Pull request by jonathanspw**

### CODE REVIEW

**Comment by jonathanspw** [comment]
Please do not merge.

**Comment by jonathanspw** [comment]
This is ready to go!

**Comment by jonathanspw** [comment]
Preview URL: https://29dfabca.almalinux-org.pages.dev/amd-epyc-june2024/

**Comment by codyro** [comment]
@mattlasheboro Mind glancing over this to see if there is going to be any weird display issues (that are obvious)?

File: i18n/en.json [hunk 1]
```diff
@@ -620,5 +620,9 @@
    "AlmaLinux Lead Architect": "AlmaLinux Lead Architect",
    "AlmaLinux 9.4 and AlmaLinux 8.10 are both stable! Checkout ": "AlmaLinux 9.4 and AlmaLinux 8.10 are both stable! Checkout ",
    "AlmaLinux OS 8.10 Stable": "AlmaLinux OS 8.10 Stable",
-   " for more information!": " for more information!"
+   " for more information!": " for more information!",
+   "processor series": "processor series",
+   "Motherboard": "Motherboard",
+   "tests cover benchmarking of OS features. The result of benchmarking can be found in the PDF linked above.": "tests cover benchmarking of OS features. The result of benchmarking can be found in the PDF linked above.",
+   "The conducted tests ensure that AMD EPYC™ 4004 series CPUs and, generally for any servers based on EPYC processors, AlmaLinux OS works properly guaranteeing stability, security, and high performance.": "The conducted tests ensure that AMD EPYC™ 4004 series CPUs and, generally for any servers based on EPYC processors, AlmaLinux OS works properly guaranteeing stability, security, and high performance."
```
Comment (codyro) [inline i18n/en.json:627]: Added missing commas

> The conducted tests ensure that AMD EPYC™ 4004 series CPUs and,  generally, for any servers based on EPYC processors, AlmaLinux OS works properly, guaranteeing stability, security, and high performance.

**Review by codyro — APPROVED** [review]
Looks good. Added two commas.

**Comment by mattlasheboro** [comment]
> @mattlasheboro Mind glancing over this to see if there is going to be any weird display issues (that are obvious)?

There are some issues on small displays.  I can send over some fixes tonight if this can wait.

**Comment by jonathanspw** [comment]
> > @mattlasheboro Mind glancing over this to see if there is going to be any weird display issues (that are obvious)?
> 
> There are some issues on small displays. I can send over some fixes tonight if this can wait.

Have you had a chance to poke at this yet?

**Comment by jonathanspw** [comment]
I need to get this merged.  I still welcome the fixes you mentioned when you get a chance to work on them.

**Comment by mattlasheboro** [comment]
> I need to get this merged. I still welcome the fixes you mentioned when you get a chance to work on them.

Sounds good, the only important change was the one suggested above that fixed the button being broke on mobile.   Everything else is purely aesthetic.

**Pull request merged**